File: 백준/Silver/1978. 소수 찾기/소수 찾기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        String[] num = br.readLine().split(" ");

        int count = 0;
        for (String v : num) {
            int value = Integer.parseInt(v);
            count++;
            if (value == 2) {
                continue;
            }
            if (value == 1) {
                count--;
                continue;
            }
            for (int j = 2; j < value; j++){
                if (value % j == 0) {
                    count--;
                    break;
                }
            }
        }

        System.out.println(count);
    }
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
N = int(input())
nums = list(map(int, (input()).split()))
count = len(nums)

for n in nums:
    if n == 1:
        count -= 1
        continue
    for i in range(2, int(n**0.5)+1):
        if n % i == 0:
            count -= 1
            break

print(count)